src/facade/dragonfly_connection.cc

```diff
@@ -1987,8 +1987,8 @@ facade::ConnectionContext* Connection::cntx() {
   return cc_.get();
 }
 
-void Connection::RequestAsyncMigration(util::fb2::ProactorBase* dest) {
-  if (!migration_enabled_ || cc_ == nullptr) {
+void Connection::RequestAsyncMigration(util::fb2::ProactorBase* dest, bool force) {
```

Comment:
Just keep in mind that any migration can potentially break anything that traverses (with preemption) the connections because we first traverse them, collect a WeakPtr from them and then we do the preemptive operation. If there is a forced migration inbetween it can cause a problem.